web/src/app/pages/create-survey/survey-details/survey-details.component.ts

```diff
@@ -56,4 +65,11 @@ export class SurveyDetailsComponent implements OnInit {
       this.formGroup.controls[this.descriptionControlKey].value,
     ];
   }
+
+  @HostListener('document:mousedown', ['$event'])
+  captureClick(event: any) {
```

Comment:
@rfontanarosa Can we use the right type here? I think it's [MouseEvent](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent) ?